tests/appsec/test_asm_standalone.py

```diff
@@ -37,14 +37,21 @@ def setup_no_appsec_upstream__no_attack__is_kept_with_priority_1__from_minus_1(s
 
     def test_no_appsec_upstream__no_attack__is_kept_with_priority_1__from_minus_1(self):
         spans_checked = 0
+        i = 0
         for data, trace, span in interfaces.library.get_spans(request=self.r):
+            # The propagated tags are set on the first span of every trace chunk
+            if i == 0:
+                first_trace = trace[0]
+                assert "_dd.p.appsec" not in first_trace["meta"]
+                assert "_dd.p.other" in first_trace["meta"]
+                assert first_trace["trace_id"] == 1212121212121212121
```

Comment:
The sampling priority is also been checked in the first span

```suggestion
                assert first_trace["trace_id"] == 1212121212121212121
                assert first_trace["metrics"]["_sampling_priority_v1"] < 2
```

Reply:
Same comment as below